src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.Development.json

```diff
@@ -1,7 +1,7 @@
 {
   "AdminUri": "localhost:5021",
   "NotifyApiKey": "change-me",
-  "NotifyInviteTemplateId": "change-me",
+  "NotifyInviteWithRolesTemplateId": "change-me",
```

Comment:
I've removed it here because it's a local setting and everything runs fine locally without it - I believe. I've left it in the Data Factory config because I've seen instances in the past where removing a variable can cause the infrastructure deploy to fail. Not sure if that would happen here, but I'm playing it safe and will instead remove all the additional config as part of EES-3428 when this work has been deployed.